course-mw/enrolment-actor/src/main/scala/org/sunbird/enrolments/CourseEnrolmentActor.scala

```diff
@@ -398,6 +399,13 @@ class CourseEnrolmentActor @Inject()(@Named("course-batch-notification-actor") c
             val progress: Int = enrolment.getOrDefault("progress", 0.asInstanceOf[AnyRef]).asInstanceOf[Int]
             enrolment.put("status", getCompletionStatus(progress, leafNodesCount).asInstanceOf[AnyRef])
             enrolment.put("completionPercentage", getCompletionPerc(progress, leafNodesCount).asInstanceOf[AnyRef])
+            val lrc_progress: String = enrolment.get(JsonKey.LRC_PROGRESS_DETAILS).asInstanceOf[String]
```

Comment:
use option and assign empty string if this is null

Reply:
updated